tasks/e2e/cypress/integration/02-cli/02-cli-commands.spec.js

```diff
@@ -13,22 +13,22 @@ describe('Check Redwood cli commands against tutorial', () => {
   after(() => {
     Cypress.config('record', true)
   })
-  it('Should run api tests successfully', () => {
-    cy.exec(`cd ${BASE_DIR}; yarn rw test api --no-watch`)
+  it.skip('Should run api tests successfully', () => {
```

Comment:
uhoh skip

Reply:
These kind of tests don't really belong in Cypress, so I've moved them over to the GitHub action. They'll be deleted, but the skips here are for my convenience whilst this is in draft.